awsx/schema-types.ts

```diff
@@ -107,11 +107,12 @@ export abstract class Image<TData = any> extends (pulumi.ComponentResource)<TDat
 }
 export interface ImageArgs {
     readonly args?: pulumi.Input<Record<string, pulumi.Input<string>>>;
+    readonly builderVersion?: BuilderVersionInputs;
     readonly cacheFrom?: pulumi.Input<pulumi.Input<string>[]>;
+    readonly context?: pulumi.Input<string>;
     readonly dockerfile?: pulumi.Input<string>;
-    readonly env?: pulumi.Input<Record<string, pulumi.Input<string>>>;
```

Comment:
What happened to env?  Isn’t this a breaking change? (And material loss of functionality?)